WLogger/Model/Log.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Log.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/10/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+import Argo
+import Curry
+
+public struct Log {
```

Comment:
I would have liked to uncomment the rest of the properties. All of them are necessary.

The issue is that the compiler complains about the expression being too complex. The suggested workaround didn't work.